pandas/core/indexes/numeric.py

```diff
@@ -1,26 +1,22 @@
 import warnings
 
 import numpy as np
+
 from pandas._libs import index as libindex
+from pandas.util._decorators import Appender, cache_readonly
```

Comment:
Isn't this supposed to be alphabetical? Should come after the following import, no?